mava/utils/make_env.py

```diff
@@ -91,10 +86,12 @@ def make_jumanji_env(
     # Create envs.
     env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
     eval_env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
-    env, eval_env = wrapper(env), wrapper(eval_env)
+    env, eval_env = wrapper(env, has_global_state=add_global_state), wrapper(
+        eval_env, has_global_state=add_global_state
+    )
 
-    env = add_optional_wrappers(env, config, add_global_state)
-    eval_env = add_optional_wrappers(eval_env, config, add_global_state)
+    env = add_optional_wrappers(env, config)
+    eval_env = add_optional_wrappers(eval_env, config)
 
     env = AutoResetWrapper(env)
     env = RecordEpisodeMetrics(env)
```

Comment:
Im not sure about this. They aren't used in the `eval_env`, which also uses `add_optional_wrappers`.

Reply:
In the current state this function is doing only one job so let's increase it's importance 😅

Reply:
Okay, that makes sense

Reply:
Unless, we just remove `add_optional_wrappers`. I mean, these two wrappers aren't really "optional" :thinking_face:

Reply:
Perhaps change the name of the function to `add_extra_wrappers ` or another more appropriate name? Otherwise we'll still have at least 7 repeated lines in every make function if we remove this fct 🤔

Reply:
I think we should just remove this function since it's only doing one thing

Reply:
Ah I see what you mean. Okay, Ill move everything to an `add_extra_wrappers` function